src/core/aws-net.ads

```diff
@@ -308,18 +311,18 @@ package AWS.Net is
    --  Returns True if the message associated with the Exception_Occurence for
    --  a Socket_Error is a "socket closed by peer".
 
-   function To_FD_Set
-     (Socket : Socket_Type;
-      Events : Wait_Event_Set;
-      Size   : Positive := 1) return FD_Set'Class;
-   --  Create appropriate socket FD set and put Socket fd there
-
    --------------------
    -- Socket FD sets --
    --------------------
 
    type FD_Set_Access is access all FD_Set'Class;
 
+   function To_FD_Set
```

Comment:
Ok with this move, but would have been better in a separate commit.

Reply:
will move back